dev/reference/tools/tikv-control.md

```diff
@@ -165,15 +165,31 @@ The `tombstone` command is usually used in circumstances where the sync-log is n
 
 In a TiKV instance, you can use this command to set the status of some Regions to Tombstone. Then when you restart the instance, those Regions are skipped. Those Regions need to have enough healthy replicas in other TiKV instances to be able to continue writing and reading through the Raft mechanism.
 
-```bash
-pd-ctl>> operator add remove-peer <region_id> <peer_id>
-$ tikv-ctl --db /path/to/tikv/db tombstone -p 127.0.0.1:2379 -r 2
-success!
-```
+Follow the two steps to set a Region to Tombstone:
+
+1. Take off the corresponding Peer of this Region on the machine in `pd-ctl`:
```

Comment:
Since `take off` has other meanings, how about using a clearer word?

```suggestion
1. Remove the corresponding Peer of this Region on the machine in `pd-ctl`:
```